piet-cairo/src/text.rs

```diff
@@ -89,29 +112,91 @@ impl fmt::Debug for CairoText {
     }
 }
 
-impl CairoFont {
-    pub(crate) fn new(family: FontFamily) -> Self {
-        CairoFont { family }
-    }
+impl CairoTextLayoutBuilder {
+    fn build_pango_attribute(attribute: AttributeWithRange) -> PangoAttribute {
+        let mut pango_attribute = match &attribute.attribute {
+            TextAttribute::FontFamily(family) => {
+                let family = family.name();
+                PangoAttribute::new_family(family).unwrap_or_else(|| {
+                    PangoAttribute::new_family(FontFamily::SERIF.name())
```

Comment:
I figured that `FontFamily::SERIF.name()` is "nicer" in the big picture than `"serif"`. Or are you referring to something else here?

Reply:
What I mean is that as written I think this only makes sense if Pango recognizes "serif" as a generic font family and finds some serif font. If it doesn't do that, we may have to go ahead and figure out what the concrete fonts should be that correspond to the generic families.

Reply:
Pango does support that (or at least appears to on my system). This is for `new_unchecked` in the case that the provided family name cannot be found I figured it would be better to fallback to *something* and still render text instead of just panicking.

Reply:
Does pango require us to set a font? I would expect it to try and find a fallback if no font was set. In any case I would expect the absolute fallback be a sans-serif, since that's the default for most (all) UI elements. 🤷

Reply:
That's a very good point :)